data/en/1_0102.json

```diff
@@ -33,42 +33,43 @@
                         "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
                     }],
                     "questions": [{
-                        "question": "Total retail turnover",
-                        "content": [{
-                            "description": "Include:",
-                            "list": [
-                                "VAT",
-                                "internet sales",
-                                "retail sales from outlets in Great Britain to customers abroad"
-                            ]
-                        }, {
-                            "description": "Exclude:",
-                            "list": [
-                                "revenue from mobile phone network commission and top-up",
-                                "sales from catering facilities used by customers",
-                                "lottery sales and commission from lottery sales",
-                                "sales of car accessories and motor vehicles",
-                                "NHS receipts"
-                            ]
-                        }]
-                    }, {
-                        "question": "Internet sales",
-                        "content": [{
-                            "description": "Include: VAT"
-                        }]
-                    }, {
-                        "question": "Significant changes to the total retail turnover",
-                        "content": [{
-                            "description": "Include:",
-                            "list": [
-                                "in-store / online promotions",
-                                "special events (e.g.  sporting events)",
-                                "calendar events (e.g.  Christmas, Easter, Bank Holiday)",
-                                "weather",
-                                "store closures/openings"
+                            "question": "Total retail turnover",
+                            "content": [{
+                                    "description": "Include:",
+                                    "list": ["VAT", "internet sales", "retail sales from outlets in Great Britain to customers abroad"]
```

Comment:
The formatting of the json seems to have changed how the lists are structured.
We should try to avoid formatting changes to the schemas as it makes it harder to see the actual changes for this PR

Reply:
Agreed. If we set up a pre-commit hook to run the json formatter then it will eradicate accidental formatting changes so nobody has to worry about it. We do this with Prettier on author and it works great.